### DISCUSSION TITLE
Broken Link

### DISCUSSION THREAD

**Discussion by Jorge4242** [six7/figma-tokens#1394] [Q&A]
What does the broken link indicator when looking at a text style mean?
<img width="287" alt="image" src="https://user-images.githubusercontent.com/116912480/198704564-5cd6c034-117e-4fdb-b362-e116a11d45e1.png">

**Comment by mgunnin** [comment]
Seeing the same thing and can't figure out why.

**Comment by mgunnin** [comment]
Just figured it out.. It's because one of the fields doesn't have an option, but still tries to set one. Mine was for "Paragraph indent".. Once I deleted it, the "disconnected icon" disappeared.

**Reply by sykang013** [reply]
Thanks!

**Comment by Jorge4242** [comment]
Awesome! Thanks

On Sun, Oct 30, 2022 at 1:24 PM Matthew Gunnin ***@***.***>
wrote:

> Just figured it out.. It's because one of the fields doesn't have an
> option, but still tries to set one. Mine was for "Paragraph indent".. Once
> I deleted it, the "disconnected icon" disappeared.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/six7/figma-tokens/discussions/1394#discussioncomment-4013421>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A337CYDTMPJYRZVWN3FC2LDWF3DNJANCNFSM6AAAAAARRJWITY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Jordan Hunter
Maven Software Development
***@***.***
